src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CipherReference.cs

```diff
@@ -41,7 +41,7 @@ internal byte[] CipherValue
 
         public override XmlElement GetXml()
         {
-            if (CacheValid) return _cachedXml;
+            if (CacheValid) return _cachedXml!;
```

Comment:
Is this `!` still required now that CacheValid declares _cachedXml can't be null?